src/display/annotation_layer.js

```diff
@@ -1942,7 +1907,7 @@ class LineAnnotationElement extends AnnotationElement {
     // trigger the popup, not the entire container.
     const data = this.data;
     const { width, height } = getRectDims(data.rect);
-    const svg = this.svgFactory.create(width, height);
+    const svg = this.svgFactory.create(width, height, /* noDims = */ true);
```

Comment:
Here, and elsewhere, please update all of these to mention `skipDims` in the comment instead.